### ISSUE TITLE
index.md is needed when creating nested directories (aka should greenwood have template defaults?)

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#120]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
Related to this test case
https://github.com/ProjectEvergreen/greenwood/blob/master/test/cli/cases/build.default.workspace-nested/build.default.workspace-nested.spec.js#L27

I guess the question / bug is: should Greenwood generate a default _index.html_ for the user?

I think the answer would be yes, but I suppose, if the user is making nested directories, wouldn't at that point having a home page make sense?

## Details
For reference, here is the error when deleting that file and running `yarn test`
```shell
  Build Greenwood With:
    Default Greenwood Configuration and Default Workspace w/ Nested Directories
      1) "before all" hook for "should create a default blog page directory"


  0 passing (6s)
  1 failing

  1) Build Greenwood With:
       Default Greenwood Configuration and Default Workspace w/ Nested Directories
         "before all" hook for "should create a default blog page directory":
     Error: the string "\u001b[31m./.greenwood/app/app.js\nModule not found: Error: Can't resolve '../index/index.js' in '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood/test/cli/cases/build.default.workspace-nested/.greenwood/app'\u001b[39m\n" was thrown, throw an Error :)
```

I think [this](https://github.com/ProjectEvergreen/greenwood/blob/master/packages/cli/templates/app-template.js#L15
) is the part that is tripping things up; the hardcoded in Greenwood's app-template 
```
import '../index/index.js';
```

Again, not sure if Greenwood should be doing this for the user, but I think that's been our general convention.

**Comment by thescientist13** [comment]
Interesting question, if using an external data source, could we do away with the need to having any local markdown?  🤔

**Comment by thescientist13** [comment]
Or we should just add something about this to the documentation and make sure it is explicit for users.

**Comment by thescientist13** [comment]
I think this is taken care of as part of #355

**Comment by thescientist13** [comment]
no longer an issue, solved as part of `v0.10.0`.
